### PR TITLE
Update OVMS images and template

### PR DESCRIPTION
**Pull request by heyselbi**
Closes: https://issues.redhat.com/browse/RHOAIENG-3469 --> addressed by update SHAs for RHOAI. OVMS ODH images will not be updated for now.
https://issues.redhat.com/browse/RHOAIENG-3355 --> addressed by the `--target_device=AUTO` line in the template
Partially closes: https://issues.redhat.com/browse/RHOAIENG-3052 --> addressed by the `--enable_metrics` line in the template
Partially addresses: https://issues.redhat.com/browse/RHOAIENG-3134 --> updating openvino_ir format to latest and adding now newly supported pytorch format

## How Has This Been Tested?

I tested this manually on a ROSA-hosted cluster. New image is working well. And all additional args are verified. Comments have been added in the relevant JIRAs with details.

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by heyselbi** [comment]
cc @Xaenalt @z103cb --> if one of you could give a thumbs up, that would be awesome.

File: manifests/overlays/modelserving/kustomization.yaml [hunk 1]
```diff
@@ -17,4 +17,4 @@ images:
     digest: sha256:e4d24fd401fd4eb89b49b4ab07e0c08389384d4a672b240e98a03ad7f9ef1c85
   - name: quay.io/opendatahub/openvino_model_server
     newName: quay.io/modh/openvino_model_server
-    digest: sha256:007304a96acd654ca5133c50990c6785464fcea44304c8a846d3279b9c83a9d4
```
Comment (heyselbi) [inline manifests/overlays/modelserving/kustomization.yaml:20]: @lucferbux given that the OVMS image is the same for KServe and MM, hopefully that wouldn't cause any issues here with replacing them?

Comment (Xaenalt) [reply manifests/overlays/modelserving/kustomization.yaml:20]: Yeah, same image is the standard case here, it's just the templates that differ

**Review by Xaenalt — APPROVED** [review]

**Review by andrewballantyne — APPROVED** [review]
Selbi and Sean have tested it. Rubber stamping.

**Review by lucferbux — APPROVED** [review]
/lgtm

**Pull request merged**